packages/react-native/ReactCommon/react/renderer/core/ShadowNode.h

```diff
@@ -252,6 +266,15 @@ class ShadowNode : public Sealable,
       const ShadowNode& sourceShadowNode,
       const Props::Shared& props);
 
+  Unshared cloneMultipleRecursive(
+      const ShadowNode& shadowNode,
+      const std::unordered_set<const ShadowNodeFamily*>& families,
```

Comment:
Let's pass `ShadowNodeFamily` as a `std::ref` instead of a pointer maybe?
```suggestion
      const std::unordered_set<const ShadowNodeFamily &>& families,
```